domains/bighead.localplayer.dev.json

```diff
@@ -0,0 +1,12 @@
+{
+    "domain": "localplayer.dev",
+    "subdomain": "bighead",
+    "owner": {
+        "repo": "https://github.com/301frequenter/register",
+        "email": "301frequenter@proton.me"
+    },
+    "record": {
+        "CNAME": "bighead.pages.dev",
```

Comment:
JSON is not valid, run it through a JSON validator and make sure all checks pass on the pull request.